test/language/generators/generator.declareMethod.shadow-parameter-catch.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ GeneratorMethod early SyntaxError when lexical declaration
+ inside generator shadows parameter name 
+es6id: 14.4.1
+author: Sam Mikes
+description: GeneratorMethod error with lexical shadowing
+negative: SyntaxError
+---*/
+
+var obj = {
+    *foo(a) {
+        try {
+        }
+        catch (e) {
+            let a = 3;
```

Comment:
I'll remove this test for now.  Originally the parameter to `catch` was named `a`, but I am unclear on whether that's legal or not, and v8 is not a suitable reference implementation for this because their let/const handling is broken.